blendergltf.py

```diff
@@ -1011,7 +1014,7 @@ def export_image(image):
     return {'image_' + image.name: export_image(image) for image in images if check_image(image)}
 
 
-def export_textures(textures):
+def export_textures(textures, images_allow_srgb):
```

Comment:
We should pass in the entire settings dictionary here to be consistent with other functions.